projects/aca-content/src/lib/dialogs/folder-details/folder-information.component.ts

```diff
@@ -0,0 +1,110 @@
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Component, DestroyRef, inject, OnInit, ViewEncapsulation } from '@angular/core';
+import { CommonModule, NgOptimizedImage } from '@angular/common';
+import { DIALOG_COMPONENT_DATA, LocalizedDatePipe, TimeAgoPipe } from '@alfresco/adf-core';
+import { JobIdBodyEntry, Node, SizeDetails } from '@alfresco/js-api';
+import { MatDividerModule } from '@angular/material/divider';
+import { TranslateModule, TranslateService } from '@ngx-translate/core';
+import { ContentService, NodesApiService } from '@alfresco/adf-content-services';
+import { concatMap, expand, first, switchMap } from 'rxjs/operators';
+import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
+import { EMPTY, timer } from 'rxjs';
+
+const MEMORY_UNIT_LIST = ['bytes', 'KB', 'MB', 'GB', 'TB'];
+
+class FolderDetails {
+  name: string;
+  size: string;
+  location: string;
+  created: Date;
+  modified: Date;
+  icon: string;
+}
+
+@Component({
+  selector: 'app-folder-info',
+  standalone: true,
+  imports: [CommonModule, MatDividerModule, TimeAgoPipe, TranslateModule, LocalizedDatePipe, NgOptimizedImage],
+  templateUrl: './folder-information.component.html',
+  styleUrls: ['./folder-information.component.scss'],
+  encapsulation: ViewEncapsulation.None,
+  host: { class: 'app-folder-info' }
+})
+export class FolderInformationComponent implements OnInit {
+  readonly contentService = inject(ContentService);
+  readonly nodesService = inject(NodesApiService);
+  readonly translateService = inject(TranslateService);
+
+  private readonly destroyRef = inject(DestroyRef);
+
+  data: Node = inject(DIALOG_COMPONENT_DATA);
+  folderDetails: FolderDetails = new FolderDetails();
+
+  ngOnInit() {
+    this.folderDetails.name = this.data.name;
+    this.folderDetails.location = this.data.path.name;
+    this.folderDetails.created = this.data.createdAt;
+    this.folderDetails.modified = this.data.modifiedAt;
+    this.folderDetails.icon = this.contentService.getNodeIcon(this.data);
+    this.folderDetails.size = this.translateService.instant('APP.FOLDER_INFO.CALCULATING');
+
+    this.nodesService
+      .initiateFolderSizeCalculation(this.data.id)
+      .pipe(
+        first(),
+        switchMap((jobIdEntry: JobIdBodyEntry) => {
```

Comment:
No need to set type in this place because it's inferred by ts: 
<img width="479" alt="image" src="https://github.com/user-attachments/assets/4851455e-20f3-413e-b3ae-f3f7d6f789b5" />

Reply:
Removed type